src/code42cli/logger.py

```diff
@@ -14,6 +21,21 @@
 ERROR_LOG_FILE_NAME = "code42_errors.log"
 
 
+def _get_formatter(output_format):
+    if output_format == FileEventsOutputFormat.JSON:
+        return FileEventDictToJSONFormatter()
+    elif output_format == FileEventsOutputFormat.CEF:
+        return FileEventDictToCEFFormatter()
+    else:
+        return FileEventDictToRawJSONFormatter()
+
+
+def _init_logger(logger, handler, output_format):
+    formatter = _get_formatter(output_format)
+    logger.setLevel(logging.INFO)
+    return add_handler_to_logger(logger, handler, formatter)
+
+
```

Comment:
Can you delete `get_logger_for_stdout()` while you are at it? We don't use that anymore